### PR TITLE
Notify that SecureBootModel should be disabled to update on 14.4+

### PR DESCRIPTION
**Pull request by lirilarp**
I probably need to reword this, let me know

### CODE REVIEW

**Review by kasper9n — APPROVED** [review]

**Review by Krazy-Killa — CHANGES_REQUESTED** [review]
This isn't necessarily a bug, but an additional security measure added by Apple during the update process.

From what I understand, the updater is requesting information from the T2 chip (on compatible SMBIOSes that have a T2 chip), and it is suppose to respond with a proper decrypt key to continue the process, but since our Hacks do not have this chip, this process fails and results in the updater failing the update and rebooting.

Probably best to reword this as "Due to increased security measures in Sonoma and newer, it is advised to set `SecureBootModel` to `Disabled` until the update is completed."  Otherwise this looks good.

**Comment by byhal** [comment]
Thank you 🙂
Will make requested changes
